### DISCUSSION TITLE
Can py-xbrl parse a xbrl gl instance?

### DISCUSSION THREAD

**Discussion by 1391098** [manusimidt/py-xbrl#150] [Q&A]
I tried to use py-xbrl to parse a xbrl gl instnce. I failed.

**Comment by manusimidt** [comment]
Hello, 
No, currently I only focused and tested on normal XBRL or iXBRL instance documents and not XBRL GL.
But I would be interested where and how it is already used :)

**Reply by 1391098** [reply]
Thanks.

http://www.xbrlgl.org.cn/index_english.html
